azkaban-web-server/src/main/java/cloudflow/docs/ExecutionDetailSample.md

```diff
@@ -0,0 +1,202 @@
+{
+  "executionId": 3,
+  "submitUser": "azkaban",
+  "submitTime": 1581552354517,
+  "concurrentOption": "skip",
+  "failureAction": "FINISH_CURRENTLY_RUNNING",
+  "experimentId": "none",
+  "description": "none",
+  "previousFlowExecutionId": "none",
+  "notifyFailureFirst": false,
+  "notifyFailureLast": false,
+  "rootFlow": {
+    "nodeId": "flowPriority2",
+    "startTime": 1581552355458,
+    "endTime": -1,
+    "status": "RUNNING",
+    "flowInfo": {
+      "flowId": "flowPriority2",
+      "flowName": "none",
+      "flowVersion": "1"
+    },
+    "nodeList": [
+      {
+        "nodeId": "jobD",
+        "startTime": -1,
+        "endTime": -1,
+        "status": "READY",
+        "flowInfo": null,
+        "nodeList": null
+      },
+      {
+        "nodeId": "jobE",
+        "startTime": -1,
+        "endTime": -1,
+        "status": "READY",
+        "flowInfo": null,
+        "nodeList": null
+      },
+      {
+        "nodeId": "emb_1",
+        "startTime": 1581552355459,
+        "endTime": 1581552355697,
+        "status": "SUCCEEDED",
+        "flowInfo": {
+          "flowId": "flowPriority2:emb_1",
+          "flowName": "none",
+          "flowVersion": "1"
+        },
+        "nodeList": [
+          {
+            "nodeId": "jobE1B",
+            "startTime": 1581552355634,
+            "endTime": 1581552355679,
+            "status": "SUCCEEDED",
+            "flowInfo": null,
+            "nodeList": null
+          },
+          {
+            "nodeId": "jobE1A",
+            "startTime": 1581552355544,
+            "endTime": 1581552355610,
+            "status": "SUCCEEDED",
+            "flowInfo": null,
+            "nodeList": null
+          }
+        ]
+      },
+      {
+        "nodeId": "emb_2",
+        "startTime": 1581552355542,
+        "endTime": -1,
+        "status": "RUNNING",
+        "flowInfo": {
+          "flowId": "flowPriority2:emb_2",
+          "flowName": "none",
+          "flowVersion": "1"
+        },
+        "nodeList": [
+          {
+            "nodeId": "jobE2C",
+            "startTime": -1,
+            "endTime": -1,
+            "status": "READY",
+            "flowInfo": null,
+            "nodeList": null
+          },
+          {
+            "nodeId": "jobE2B",
+            "startTime": 1581552355661,
+            "endTime": 1581552355697,
+            "status": "SUCCEEDED",
+            "flowInfo": null,
+            "nodeList": null
+          },
+          {
+            "nodeId": "emb_3",
+            "startTime": 1581552355718,
+            "endTime": -1,
+            "status": "RUNNING",
+            "flowInfo": {
+              "flowId": "flowPriority2:emb_2:emb_3",
+              "flowName": "none",
```

Comment:
`flowId` should be something like "1a2b3c4" and flowName is "emb_3" in this case.

Reply:
Where is the `"1a2b3c4"` like information ? I don't see it in the Flow.java or the `project_flows` table in the db. 
I began to suspect the `flowId` and `flowName` and one of them can be removed.

Reply:
This may change when there is a new table for flows. Have a `todo` for it in ExecutionServiceImpl.